### PR TITLE
Add multiple outputs (stdout + stderr) and support increase task runtime (to simulate long running processes)

### PR DESCRIPTION
**Pull request by mlaszloffy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,69 @@
...
+ 
+ <<<
+     set -euo pipefail
+     # Output n lines to stderr
+     for (( i = 1; i <= ~{n}; i++ )) ; do
+       echo "This is a place holder stdout line ${i}"
+       echo "This is a place holder stderr line ${i}" 1>&2
+     done
+     exit ~{exitCode}
+   >>>
```
Comment (avarsava) [inline README.md:64]: I don't really understand this section? What is WORKFLOW workflow?

File: empty.wdl [hunk 2]
```diff
@@ -7,6 +7,7 @@ workflow empty {
     Int n = 10
   }
   parameter_meta {
+    dummyInput: "File from Vidarr"
```
Comment (avarsava) [inline empty.wdl:10]: It's not from vidarr if you run the command at README:15, is it?

**Review by Christopher-Ho — COMMENTED** [review]
Would these changes be included alongside https://github.com/oicr-gsi/empty/pull/3 or would that PR branch only be used testing?

File: README.md (hunk 1 above)
Comment (mlaszloffy) [reply README.md:64]: This is part of commands.txt that was added. I'll clean this up.

**Review by Christopher-Ho — APPROVED** [review]

**Review by avarsava — APPROVED** [review]

**Pull request merged**